integration-cli/docker_cli_events_unix_test.go

```diff
@@ -396,35 +396,28 @@ func (s *DockerDaemonSuite) TestDaemonEvents(c *testing.T) {
 	defer os.Remove(configFilePath)
 
 	daemonConfig := `{"labels":["foo=bar"]}`
-	fmt.Fprintf(configFile, "%s", daemonConfig)
-	configFile.Close()
+	_, err = fmt.Fprintf(configFile, "%s", daemonConfig)
```

Comment:
nit (no need to update for this): lol, might as well just do `configFile.Write([]byte(daemonConfig))`

Reply:
actually most of these should just be `ioutil.WriteFile` but whatever

Reply:
yeah, was thinking about that when writing, guess I was lazy 😂

Reply:
No worries